sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/README.md

```diff
@@ -68,6 +68,7 @@ Configuration Store Authentication
 Name | Description | Required | Default
 ---|---|---|---
 spring.cloud.azure.appconfiguration.stores[0].endpoint | When the endpoint of an App Configuration store is specified, a managed identity or a token credential provided using `AppConfigCredentialProvider` will be used to connect to the App Configuration service. An `IllegalArgumentException` will be thrown if the endpoint and connection-string are specified at the same time. | Conditional | null
+spring.cloud.azure.appconfiguration.stores[0].endpoints | When a list of endpoint of an App Configuration store is specified, they will be treated as replications of the same store, a managed identity or a token credential provided using `AppConfigCredentialProvider` will be used to connect to the App Configuration service. Relpicas should be listed in priority order of connection. An `IllegalArgumentException` will be thrown if multiple authentication methods are provided. | Conditional | null
```

Comment:
>When a list of endpoint of an App Configuration store is specified, they will be treated as replications of the same store [](http://example.com/codeflow?start=58&length=122)

Suggest to change it to:

When multiple replica endpoints of an App Configuration store are specified, a managed identity .....